apis/kueue/webhooks/clusterqueue_webhook.go

```diff
@@ -35,10 +35,6 @@ import (
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1alpha2"
 )
 
-var (
-	queueingStrategies = sets.NewString(string(kueue.StrictFIFO), string(kueue.BestEffortFIFO))
```

Comment:
add an integration test to make sure that using a different value or empty fails.